packages/react-components/style/output-components-style.css

```diff
@@ -237,6 +237,7 @@ canvas {
 
 .table-tree td.selected {
     background-color: var(--theia-selection-background) !important;
+    width: 500px;
```

Comment:
Due to the hardcoded width, there's a difference in hover and selection styles. Could you apply the selection style to the entire row instead of the leftmost cell:
![Screen Shot 2023-02-21 at 3 26 35 PM](https://user-images.githubusercontent.com/92893187/220462410-eb9c8bee-62ec-4caf-af20-001a07327348.png)